src/EnergyPlus/VariableSpeedCoils.cc

```diff
@@ -4576,6 +4576,7 @@ namespace VariableSpeedCoils {
         state.dataVariableSpeedCoils->VarSpeedCoil(DXCoilNum).BasinHeaterConsumption = 0.0;
         state.dataVariableSpeedCoils->VarSpeedCoil(DXCoilNum).EvapCondPumpElecConsumption = 0.0;
         state.dataVariableSpeedCoils->VarSpeedCoil(DXCoilNum).CrankcaseHeaterPower = 0.0;
+        state.dataVariableSpeedCoils->VarSpeedCoil(DXCoilNum).DefrostPower = 0.0;
         state.dataVariableSpeedCoils->VarSpeedCoil(DXCoilNum).DefrostConsumption = 0.0;
```

Comment:
See the explanation in the top comment of the PR about the root cause of the problem and how it gets fixed:
https://github.com/NREL/EnergyPlus/pull/10109#issue-1801050600

Reply:
OK, I agree with the description in the issue description, and you added the reset to Step 1 in Init. What I don't see here is the line that was deleted in Step 2, in Calc, that is no longer needed, right?